src/index.js

```diff
@@ -1 +1,43 @@
-import './style.css';
\ No newline at end of file
+import { } from 'lodash';
+import './style.css';
+import { getData, scoresUrl, display } from './api';
+
+const name = document.querySelector('.name');
+const score = document.querySelector('.score');
+const refresh = document.querySelector('.ref-btn');
+const form = document.forms['form-dom'];
+
+const reload = () => {
+  window.location.reload();
+};
+
+const add = (e) => {
+  e.preventDefault();
+  fetch(scoresUrl, {
+    method: 'POST',
+    mode: 'cors',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name.value,
+      score: Number(score.value),
+    }),
+  })
+    .then((response) => response.json())
+    .then(async () => {
+      const allScores = await getData();
+      name.value = '';
+      score.value = '';
+      display(allScores.result);
+    });
+  // reload();
```

Comment:
- [ ] Please remove commented pieces of code from your project, it makes the codebase messy and unprofessional. You can use git history if you need to look back into a previous version of the application.